apps/www/components/ExampleCard.tsx

```diff
@@ -1,84 +1,44 @@
-import { Button, Divider, IconArrowUpRight, IconGitHub, IconTriangle, Space } from 'ui'
+import { Button, IconArrowUpRight } from 'ui'
 import Link from 'next/link'
 import Image from 'next/image'
 
 function ExampleCard(props: any) {
   return (
-    <>
-      <div
-        className="bg-surface-100
-          border-default
-          flex
-          h-40
-          flex-col
-          justify-between rounded rounded-b-none
-          border
-          border-t border-r border-l p-5
-          "
-      >
-        <div className="mb-4">
-          <h4 className="h6">{props.title}</h4>
-          <p className="p text-sm">{props.description}</p>
-          <div className="flex items-center">
-            <div className="relative border border-default inline !w-6 !h-6 rounded-full overflow-hidden">
-              <Image
-                src={props.author_img}
-                alt={props.author + ' GitHub profile picture'}
-                layout="fill"
-                objectFit="cover"
-              />
-            </div>
-            <span className="text-foreground ml-2 text-sm">{props.author}</span>
+    <div className="relative rounded-xl bg-scale-400 from-scale-800 to-scale-800 p-px transition-all shadow-md h-full">
+      <div className="relative h-full z-10 rounded-xl bg-scale-200 dark:bg-scale-300 overflow-hidden transition-all text-scale-1100 flex flex-col p-5">
+        <div className="mb-4 lg:mb-8 flex-1">
+          <h2 className="text text-lg font-medium">{props.title}</h2>
+          <div className="my-2 block">
+            <p className="text-light">{props.description}</p>
           </div>
         </div>
-      </div>
-      <Divider light />
-      <div>
-        <div
-          className="
-          bg-surface-200
-          border-default flex
-          flex-col justify-between rounded rounded-t-none
-          border
-          border-b border-r border-l
-          border-t-0 p-5"
-        >
-          <Link
-            href={props.repo_url}
-            as={props.repo_url}
-            className="text-foreground-light hover:text-foreground flex flex-row items-center text-sm"
-            target="_blank"
-          >
-            <span>{props.repo_name}</span>
-            <span className="ml-1 inline-block">
-              <IconGitHub size={14} />
-            </span>
+        <div className="flex items-start gap-2">
+          <Link href={props.repo_url}>
+            <a target="_blank" tabIndex={-1}>
+              <Button size="tiny" type="default" iconRight={<IconArrowUpRight />}>
+                View Code
+              </Button>
+            </a>
```

Comment:
With next13 now next/link `<Link>` component is self contained, you don't need the inner `<a>` tag.
So you can remove the <a> tag. Also, the <Button> component should be outside using `asChild` prop.

Like this:
```tsx
<Button size="tiny" type="default" iconRight={<IconArrowUpRight />} asChild>
  <Link href={props.repo_url} target="_blank">
    View Code
  </Link>
</Button>
```